src/service/stripe.js

```diff
@@ -128,14 +128,16 @@ export function postCharge (account: string, token: string, amount: number, curr
 
   return api
   .post('/charges')
-  .set('Stripe-Account', account)
   .type('form')
   .send({
     amount: cut['total'],
-    application_fee: cut['elementary'],
     currency,
     description,
-    source: token
+    source: token,
+    destination: {
+      account,
```

Comment:
It's an es6 feature. If the key is the same as a variable name you can exclude it. It's pretty much the same as `account: account`